.github/workflows/chart-release.yaml

```diff
@@ -0,0 +1,55 @@
+name: Release Charts
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  build:
+    name: Publish Helm Chart
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    env:
+      CHART_NAME: bioscfg
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v4
+        with:
+          path: ${{ env.CHART_NAME }}
+          fetch-depth: 0
+      -
+        name: Checkout Helm chart Repo
+        uses: actions/checkout@v4
+        with:
+          repository: metal-toolbox/${{ env.CHART_NAME }}
+          path: helm-charts
+          ref: gh-pages
+          fetch-depth: 0
```

Comment:
Am I missing something? These look like the same step.

Reply:
Yes, but what branch is it pulling?

We are going to push to that branch (gh-pages). then those tarball helmcharts are available to be pulled.

check out flipflop's for reference: https://github.com/metal-toolbox/flipflop/tree/gh-pages